sites/brentter.com.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Everything's Better With Brentter'
```

Comment:
Hey @brentter — looks like you either need to use double quotes on this value or escape the apostrophe in "Everything’s".

Reply:
Thanks, I didn't even think about that. Also I mistyped in the commit, I didn't remove the comma, I removed the apostrophe. Sorry, should be good now